js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts

```diff
@@ -49,9 +49,12 @@ describe("Realm settings general tab tests", () => {
     sidebarPage.waitForPageLoad();
 
     // Disable realm
-    realmSettingsPage.toggleSwitch(`${realmName}-switch`);
+    const loadName = `load-disabled-${uuid()}`;
+    cy.intercept({ path: "/admin/realms/*", times: 1 }).as(loadName);
```

Comment:
Since we await the same request in `toggleSwitch()`, should this not be placed after the line below?

Reply:
We do not await it in `toggleSwitch` due to the `false` parameter

Reply:
In that case, would it not be possible to remove this code and let the `toggleSwitch()` method handle this intercept?

Reply:
Got it, ok